pydis_site/apps/api/models/bot/reminder.py

```diff
@@ -15,6 +15,12 @@ class Reminder(ModelReprMixin, models.Model):
             "If not, it has been sent out to the user."
         )
     )
+    jump_url = models.CharField(
+        max_length=88,
```

Comment:
What's your reason for `88` here? Is it based on existing URLs Discord generates? If so, this means that this may break in the future: snowflake IDs are integer representations of 64-bit numbers. In base10 form, they're currently one digit shorter than they could theoretically become. I'd rather have us have some margin here. (Although I haven't computed if they will get larger realistically given the time frame we live in.)

Reply:
From my calculations the first bit will be 1 when we will reach the year 71745, so I think we are pretty safe :) but do you still want to me to add a little margin of something like 2 characters?

Reply:
Yeah, that probably doesn't really matter then. (Although I think it's the year 54525, since it's not the last bit, but another digit in base10, but same thing.)